src/auro-accordion.js

```diff
@@ -124,31 +141,25 @@ export class AuroAccordion extends LitElement {
 
     const buttonClasses = {
       "trigger": true,
-      "iconRight": this.getAttribute('chevron') === 'right'
+      "iconRight": this.getAttribute('chevron') === 'right',
+      "sm": this.getAttribute('variant') === 'sm',
+      "lg": this.getAttribute('variant') === 'lg',
     }
 
-    const chevronHtml = this.getAttribute('chevron') === 'none'
-      ? html``
-      : html`
-        <div class="iconWrapper" part="chevron" slot="icon">
-          ${this.generateIconHtml(this.expanded ? chevronUp.svg : chevronDown.svg)}
-        </div>
-      `;
-
     return html`
       <div class="componentWrapper" part="accordion">
-        <auro-accordionbutton
+        <auro-accordion-button
           ?fluid="${this.emphasis}"
           class="${classMap(buttonClasses)}"
           id="accordionTrigger"
           aria-controls="accordionContent"
           aria-expanded="${this.expanded}"
           @click="${this.toggle}"
           part="trigger">
-
-          ${chevronHtml}
+          <${this.iconTag} category="interface" name="chevron-down" customSize customColor ?hidden="${this.expanded}"></${this.iconTag}>
+          <${this.iconTag} category="interface" name="chevron-up" customSize customColor ?hidden="${!this.expanded}"></${this.iconTag}>
```

Comment:
I recommend reverting this and NOT depending on the CDN to render the icons. If there is a failure that we can not address, this will cause problems.